src/Catty/Helpers/Util/Util.swift

```diff
@@ -95,4 +95,13 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         let platformVersionWithoutPatch = "\(major).\(minor)" as String
         return platformVersionWithoutPatch
     }
+
+    class func askUserForLandscapeMode() -> Bool {
```

Comment:
It is probably not the best idea to implement this function in the `Util`. Since there will be user interaction you will for sure need at least one completion block, so you will need to largely refactor this method anyhow. That's why I would recommend to completely remove it for now. Maybe in a separate ticket you can come up with some ideas on how to avoid code duplication for adding a project (`MyProjectsViewController` and `CatrobatTableViewController`). Maybe a common protocol with an extension? You could then implement the user interaction in this extension directly.